common/src/main/resources/db/changelog/v2024/misc_opt_out_attribution.sql

```diff
@@ -42,5 +42,47 @@ end;
 $$;
 -- /******* END ******/
 
+DROP procedure public.insert_new_current_mbi(); --obsolete changed MBI update from insert_new_current_mbi to another
+-- view below due to time out issues and to address AB2D-6051
+
+CREATE OR REPLACE Procedure  proc_insert_mbi_to_table(
```

Comment:
I am not sure if this matters, but there appears to be an extra space between Procedure and proc_insert_mbi_to_table(

Reply:
Fixed it , thank you